### ISSUE TITLE
Lora mesh chat?

### ISSUE THREAD

**Issue by T00ManySecrets** [bhaskarsdose/Lora-Mesh#1]
Im trying to make something like this https://github.com/TheNico14/LoRaMessenger
But i need for every node to be a mesh node that retransmits  all messages. 

How would i go about using your mesh system to do this?

**Comment by bhaskarsdose** [comment]
yes you can use this system, you just need to add OLED display for which you can use u8g2
